src/components/DataTable/index.tsx

```diff
@@ -100,7 +99,7 @@ export const DataTable: React.FC<DataTableProps> = ({ data, columns, renderRowSu
 
   return (
     <TableContainer>
-      <Table $hasSubRow={renderRowSubComponent} {...getTableProps()}>
+      <Table $hasSubRow={renderRowSubComponent !== undefined} {...getTableProps()}>
```

Comment:
would this have the same effect?
```
      <Table $hasSubRow={!!renderRowSubComponent} {...getTableProps()}>
```

Reply:
it would, I just default to a more explicit style because with `!!` it's harder to tell what kind of false-ness you're trying to check, e.g. `null`, `undefined`, `0`, `""`, `NaN`, ...